compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/symbols/BallerinaErrorTypeSymbol.java

```diff
@@ -66,7 +98,7 @@ public String signature() {
         }
 
         ModuleID moduleID = this.getModule().get().id();
-        if ("lang.annotations".equals(moduleID.moduleName()) && "ballerina".equals(moduleID.orgName())) {
+        if (moduleID.moduleName().startsWith("lang") && "ballerina".equals(moduleID.orgName())) {
```

Comment:
Use `lang.` as the `startsWith` parameter.

Reply:
Corrected.